src/webgpu/api/validation/state/device_lost/destroy.spec.ts

```diff
@@ -406,3 +408,136 @@ Tests creating query sets on destroyed device fails.
       t.device.createQuerySet({ type, count: 4 });
     });
   });
+
+g.test('command,copyBufferToBuffer')
+  .desc(
+    `
+Tests copyBufferToBuffer command on destroyed device fails.
+  - Tests that finishing encoding fails on destroyed device
+  - Tests that submitting command fails on destroyed device
+  `
+  )
+  .unimplemented();
+
+g.test('command,copyBufferToTexture,uncompressed_format')
+  .desc(
+    `
+Tests copyBufferToTexture command with various uncompressed formats on destroyed device fails
+  - Tests that finishing encoding fails on destroyed device
+  - Tests that submitting command fails on destroyed device
+  `
+  )
+  .unimplemented();
```

Comment:
I don't think we need any variations on the formats; it would be pretty weird for an implementation to pass for one format and fail for another format. Testing with just one format would be OK.

Reply:
Done.